README.md

```diff
@@ -1 +1,189 @@
-# Hello
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+  -- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello-microverse] <a name="Hello-microverse"></a>
+
+**[Hello-microverse]** this was mini-projects that upkill to create a feature branch and test linters in order to have professional-looking code.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <summary>Javascript runtime environment</summary>
+  <ul>
+    <li><a href="https://nodejs.org/en/">Node JS</a></li>
+  </ul>
+
+  <summary>Version control</summary>
+  <ul>
+    <li><a href="github.com">Git Hub</a></li>
+  </ul>
+</details
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Creating your first "Hello-microverse" project
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-A Git hub account
+<br>
+-Node JS
+<br>
+-Visual Studio Code or any other code editor of your preference
+<br>
+-web browser
+
+### Setup
+
+Clone this repository to your desired folder:<br>
+Advantages of Linters:<br>
+
+1: improves readability-it will be easy to read and make adjustment if required easily.<br>
+2:removes silly errors before execution and code review.<br>
+3:Makes code look like written by a single person.<br>
+4:Spreads awareness and ownership over code quality<br>
+5:Having more secure and performant code.<br>
```

Comment:
- [ ] You have done a great job to ensure that you direct the visitors to clone the repository into their folder. However this may not be sufficient. Lets consider guiding the visitors on use of the command lines for example `cd myfolder` or more. This will help the visitor have a clear understanding of the project setup ☺️.
- [ ] In addition, I think it would be a good practice if the advantages of the linters are removed since they are not part of the project setup.